janusgraph-cassandra/src/main/java/org/janusgraph/diskstorage/cassandra/astyanax/AstyanaxStoreManager.java

```diff
@@ -69,6 +69,7 @@
 import static org.janusgraph.diskstorage.cassandra.CassandraTransaction.getTx;
 
 @PreInitializeConfigOptions
+@Deprecated
```

Comment:
Isn't the usual approach when deprecating something to also link to the recommended alternative and maybe to add since which version it is deprecated, together with when it will be removed? (I know the second aspect at least from TinkerPop).

Reply:
Normally shouldn't use this in your code, so i though to mark it, for expert users and leave it without a comment.